packages/myforestplot.yml

```diff
@@ -0,0 +1,6 @@
+repo: toshiakiasakura/myforestplot
+section: domain specific libraries
```

Comment:
Ah, the reason why I created this packge is visualizing risk ratios in a forest plot. Maybe my knowledge is limited, but I feel a forest plot is major in epidemiology studies (and also in social sciences?), so that I just thought a forest plot package is epimiology/social science specific library.
 
 After your suggestion, yes, I agree with that. "plot types" category is more suitable for a package only creating a forestplot. No, problem.